### ISSUE TITLE
example.py error

### ISSUE THREAD

**Issue by robmarkcole** [oseiskar/simdkalman#2]
On running the example, I receive the error:
```
ValueError: ("Size of label '%s' for operand %d does not match previous terms.", 'B', 1)
```

**Comment by oseiskar** [comment]
Thanks for reporting :+1:. This is caused by numpy 1.14, which somehow seems to break the `einsum` function. I'll take a closer look. In the meantime, you can test this package by downgrading numpy to version 1.13 (`pip install numpy==1.13`)

**Comment by oseiskar** [comment]
This was caused by https://github.com/numpy/numpy/issues/10343

**Comment by oseiskar** [comment]
Fixed and released 0.9.3
